Arriba/Arriba.Test/Model/IpRangeColumnTests.cs

```diff
@@ -54,5 +57,75 @@ private string ParseAndToString(string ipRange)
 
             return "";
         }
+
+        [TestMethod]
+        public void IpRangeColumn_Basic()
+        {
```

Comment:
I think more obvious placeholders here like "0.0.0.0" would be more clear that the input doesn't matter and its just running basic tests.